lua/telescope/_extensions/file_browser/make_entry.lua

```diff
@@ -12,6 +12,7 @@ local make_entry = function(opts)
     local hl_group
     -- mt.cwd can change due to caching and traversal
     opts.cwd = mt.cwd
+    opts.path_display = { ["truncate"] = 3 }
```

Comment:
How would the user now set `path_display` if they for some reason would want to have it otherwise? One reason here I think what's worth thinking about is `path_display` in the context of the `folder_browser` and whether we can improve there.